cobrawap/__main__.py

```diff
@@ -323,35 +323,36 @@ def run_stage(profile=None, stage=None, extra_args=None, **kwargs):
     # lookup stage input file
     pipeline_config_path = config_path / 'configs' / 'config.yaml'
     config_dict = load_config_file(pipeline_config_path)
-    stage_idx = locate_str_in_list(config_dict['STAGES'], stage)
-    if stage_idx is None:
+    stage_idx_local = locate_str_in_list(config_dict['STAGES'], stage)
```

Comment:
keep the variable name as `stage_idx`

Reply:
It's important to differentiate among the stage index in the custom-defined list of stages, and the larger one present in the `~/.cobrawap/config` file.

Reply:
There is only one stage_idx in use here (the "local" one), so there is no need to differentiate

Reply:
Yes, but using "talking" names for variables helps... We can use "stage_idx" as it used to be, or "user_stage_idx", or any other meaningful suggestion...